### PR TITLE
Add USSD SMPP processor for 6D (Six DEE) USSD gateway.

### PR DESCRIPTION
**Pull request by hodgestar**
The plan is to start with a copy of the Mica processors and adjust as needed.

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

**Comment by jerith** [comment]
:+1:, although I'm guessing this will need real-world testing.

**Comment by hodgestar** [comment]
Tx, and yes, I expect surprises once we try this in the wild.

**Comment by hodgestar** [comment]
Oh, I should have waited for Travis. :/

File: vumi/transports/smpp/processors/sixdee.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+def make_vumi_session_identifier(msisdn, sixdee_session_identifier):
+    return '%s+%s' % (msisdn, sixdee_session_identifier)
+
+
+class DeliverShortMessageProcessorConfig(
+        default.DeliverShortMessageProcessorConfig):
+
+    max_session_length = ConfigInt(
+        'Maximum length a USSD sessions data is to be kept for in seconds.',
+        default=60 * 3, static=True)
```
Comment (smn) [inline vumi/transports/smpp/processors/sixdee.py]: We may want to revisit this default with this new gateway. We've been surprised before.

**Pull request merged**